docs/how_to_contribute.md

```diff
@@ -0,0 +1,53 @@
+# How to contribute
+
+Swupd is open for code contributions. Nevertheless, if you are planning to propose a new feature, it's suggested that you file an issue first so we can discuss the feature idea and the proposed implementation. This way we can avoid future conflicts with other planned features.
+
+When code your code is ready, open a pull request to have your patches reviewed. Patches are only going to be merged after being reviewed and approved by at least 2 maintainers (with some exceptions for trivial patches), all functional tests passes and enough tests are added to test the proposed functionality or bug fix.
```

Comment:
I think there is an extra "code" word in this phrase: 'When code your code is ready,"